src/core/schema.ts

```diff
@@ -24,6 +24,8 @@ export const defaultSchema: Schema = {
     right: "ArrowRight",
     home: "Home",
     end: "End",
+    page_up: "PageUp",
+    page_left: "PageLeft",
```

Comment:
Should this be `page_down: PageDown`?

Reply:
Yes it should. Could you PR? Thanks for catching!

Reply:
#678